### PR TITLE
Refactor the code

### PR DESCRIPTION
**Pull request by troublemagnet**
Before writing style tests, I thought it was a good idea to refactor the code.

The pylint score increased from 4.72->5.87.
Apart from that pep8 now reports only 4 line too long errors in settings.py (line lengths being ~81-85 characters long).

### CODE REVIEW

**Review by gedankenstuecke — CHANGES_REQUESTED** [review]
Thanks for the great styling!  👍

I would recommend leaving the migrations untouched though, as these are generated automatically by Django and all future migrations would have to be touched again this isn't really sustainable. Could you undo those changes?

Generally I think in the future we'd like to use `flake8` for checking the style (for consistency as all other Open Humans projects use it too). With `flake8` we can just write a `.flake8` file in the root of the repository and add `migrations` to the exclude list to make sure that these will not throw errors. See http://flake8.pycqa.org/en/latest/user/configuration.html for the details on how to configure flake8.

**Comment by troublemagnet** [comment]
Sure. I'll that sound good. I'll write a .flake8 file. Also should I add the flake8 in the travis build to ensure that the code base is standardized always?

**Comment by gedankenstuecke** [comment]
Thanks, I'm currently looking into the integration with a CI!

File: manage.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "oh_data_uploader.settings")
```
Comment (gedankenstuecke) [inline manage.py]: Please undo this one too. `manage.py` is created by `Django` right away and should remain untouched. Sorry for not noticing this one earlier!

File: open_humans/migrations/0001_initial.py [hunk 2]
```diff
@@ -17,12 +17,14 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='OpenHumansMember',
             fields=[
-                ('oh_id', models.CharField(max_length=16, primary_key=True, serialize=False, unique=True)),
```
Comment (gedankenstuecke) [inline open_humans/migrations/0001_initial.py]: This one also still has changes :)

**Review by gedankenstuecke — CHANGES_REQUESTED** [review]
Either I'm looking at old code or there are still some changed standard-files. :)

File: manage.py (hunk 1 above)
Comment (troublemagnet) [reply manage.py]: Yes, I'm still in the process of doing it. :sweat_smile:

**Comment by troublemagnet** [comment]
@gedankenstuecke  Tests have passed and it's ready for review now,

**Review by gedankenstuecke — APPROVED** [review]
Thanks, this looks good! With #35 I'm adding [Hound](https://houndci.com) as a Continuous Integration that will check all pull requests for style guide violations!

**Pull request merged**